manifests/init.pp

```diff
@@ -49,10 +50,22 @@
     provider => 'gem',
   }
 
+  if (is_array($scout_roles)) {
+    $scout_roles_config = inline_template('<% @scout_roles.size.to_i %>') ? {
```

Comment:
I thought so - but reconsidered as at the time I was unsure of the best approach to not join on ',' when there's only a single element in the array.   Didn't want the module to pass ```` -r ROLE,```` to scout in case a traliing comma breaks scout runs.